src/resource/tiles/schemas/tile.schema.ts

```diff
@@ -0,0 +1,32 @@
+import { Schema, Prop, SchemaFactory } from "@nestjs/mongoose";
+import { Document } from "mongoose";
+
+/* maps our cat class to our mongoDB collection of the same name */
+@Schema()
+export class Tile extends Document {
+  @Prop({ type: String, required: true })
```

Comment:
what is the default value for `required`?

Reply:
this value has to be given by the user, otherwise, this data cannot save to DB.

Reply:
Eason, my question was the default value for `required`, you set it true which means the default value should be `false`. Is my understanding correct? (if the default value is true, then you don't need to set it explicitly)

Reply:
yes, the default value is false.